blog/_posts/2022-05-30-long-term-compatibility-plans.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: blog-detail
+post-type: blog
+by: Paweł Marks, VirtusLab
+title: Long-term compatibility plans for Scala 3
+---
+
+Scala 3 has stronger compatibility guarantees than Scala 2 ever had. That allows us to improve the language at a much faster pace than it was possible before 3.0. However, with the current scheme, when you bump the dependencies of your project, you may also have to bump the version of Scala used by the project. After an extended evaluation, we have decided to solve them by creating a new kind of Scala release - Scala LTS, that will be supported and receive updates for at least three years after the initial release.
```

Comment:
"we have decided to solve them" - sorry, maybe I'm just blind, but what does "them" refer to?

Reply:
😅 Refactoring leftover